ome-dundeeomero.yml

```diff
@@ -254,9 +248,12 @@
     #omero_server_db_dumpdir_parent: /tmp
     #omero_server_db_dumpdir_name: nightly-pg_dump_omero.dir
 
+    ice_version: "3.6"
+    ice_install_devel: False
+    ice_python_wheel: "https://github.com/ome/zeroc-ice-py-centos7/releases/download/0.1.0/zeroc_ice-3.6.4-cp27-cp27mu-linux_x86_64.whl"
```

Comment:
Outwith this PR, it seems unfortunate to need to specify this URL everywhere.

Reply:
FYI I don't think these changes are strictly necessary, I think the problem may have been due to a change in the zeroc yum repo. Running this in check mode yesterday showed a change to the yum repo, and since this was in check mode the change wouldn't have taken effect, which means the package wouldn't be found. In the absence of a molecule test it's difficult to verify this.

Reply:
We should not need ice-devel on nightshade.

Reply:
We shouldn't, but if it was previously in the playbook then without a clean install (e.g. a molecule test) we can't be sure there isn't anything secretly depending on it. `ice_install_devel: True` is also the default for the role so represents the current state of the server